.github/actions/pytest_run_python/action.yml

```diff
@@ -4,7 +4,9 @@ runs:
   using: "composite"
   steps:
     - name: Python tests with pytest
-      run:
-        python -m pytest -rx -m "not parallel and python" --ignore=printers --ignore=symbolic --ignore=ndarrays
+      run: |
+        python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and python" --ignore=printers --ignore=symbolic --ignore=ndarrays
+        #python -m pytest -rx -m "xdist_incompatible and not parallel and python" --ignore=printers --ignore=symbolic --ignore=ndarrays
```

Comment:
What's the meaning of this commented line?

Reply:
If we have any python tests which are incompatible with `pytest.xdist` then this line needs uncommenting. However it cannot be uncommented now as pytest raises error code `5` if no tests are found